server/block/frosted_ice.go

```diff
@@ -0,0 +1,118 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/sound"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+// FrostedIce ...
+type FrostedIce struct {
+	solid
+
+	// Age ...
+	Age int
+}
+
+// Instrument ...
+func (FrostedIce) Instrument() sound.Instrument {
+	return sound.Chimes()
+}
+
+// BreakInfo ...
+func (p FrostedIce) BreakInfo() BreakInfo {
```

Comment:
Code styling—receiver names should be consistent, preferably `f` rather than `p` or `fi`

Reply:
Apologies, I copy pasted the entire file from an already existing block as a template and forgot about the receiver names.